schema/pod.go

```diff
@@ -83,6 +83,15 @@ func (pm *PodManifest) assertValid() error {
 	if pm.ACKind != PodManifestKind {
 		return pmKindError
 	}
+
+	// ensure volumes names are unique (unique key)
+	volNames := make(map[types.ACName]bool, len(pm.Volumes))
```

Comment:
I thought this was golang-idiomatic to make the if-has-key check more compact. The same pattern is used in other places in appc/spec, so I'd leave it as is.